.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "answer-store"]
+	path = answer-store
+	url = https://github.com/yt-project/answer-store
+	branch = yt-4.0
```

Comment:
+1 to using a separate branch for these tests. Then we can have updates for tests on master too and after the 4.0 merge we won't have annoying history conflicts. 

Maybe we'll also need a 4.0 merge PR for the answer store then, too? 😉

Reply:
Yes, I think so!